wsd/DocumentBroker.cpp

```diff
@@ -2981,10 +2983,7 @@ std::size_t DocumentBroker::countActiveSessions() const
     std::size_t count = 0;
     for (const auto& it : _sessions)
     {
-        if (it.second->isViewLoaded() && !it.second->inWaitDisconnected())
-        {
-            ++count;
-        }
+        count += it.second->isLive();
```

Comment:
Not thrilled by adding a bool to a size_t - of course it may work - but it looks highly un-readable and paranoid programmers who know there is no boolean register type may freak out (like me)

Can we have it in the conditional as before ?

Reply:
`bool` is implicitly promoted to integral types. The language rules guarantee that `true` is 1 and `false` is 0. (The opposite is also true, when coercing an integral value into a bool.) The language guarantees that no other value can be stored in a bool, so we can't end up adding arbitrary numbers here.

We can have the conditional, of course, it just looks rather verbose and unnecessary.